plugwise/smile.py

```diff
@@ -214,28 +228,8 @@ def get_all_devices(self):
 
         return devices
 
-    def get_device_data(self, dev_id):
-        """Provide device-data, based on location_id, from APPLIANCES."""
-        devices = self.get_all_devices()
-        details = devices.get(dev_id)
-        device_data = self.appliance_data(dev_id)
-
-        # Generic
-        if details["class"] == "gateway" or dev_id == self.gateway_id:
-            # Anna: outdoor_temperature only present in domain_objects
-            if "outdoor_temperature" not in device_data:
-                outdoor_temperature = self.object_value(
-                    "location", self._home_location, "outdoor_temperature"
-                )
-                if outdoor_temperature is not None:
-                    device_data["outdoor_temperature"] = outdoor_temperature
-
-            # Try to get P1 data and 2nd outdoor_temperature, when present
-            power_data = self.power_data_from_location(details["location"])
-            if power_data is not None:
-                device_data.update(power_data)
-
-        # Switching Groups
+    def get_device_data_switching_groups(self, details, device_data):
```

Comment:
Suggest to remove `get_` from the names of 4 device_data sub functions.

Reply:
👍